src/util/address.ts

```diff
@@ -1,8 +1,9 @@
 /**
- * @module util/address
+ * @module util/addressbool
  */
+import {bool} from "../types";
 
-export function isValidAddress(address: string) {
+export function isValidAddress(address: string): bool {
```

Comment:
I would just use the built-in `boolean` type here.
`bool` is used more for things that need to be serialized.

Reply:
updated in new commit